tdrs-backend/tdpservice/users/migrations/0020_ofa_system_admin_permissions.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.2.5 on 2021-08-16 14:10
+from django.db import migrations
+
+from tdpservice.users.permissions import delete_permissions_q
+
+
+def set_ofa_system_admin_permissions(apps, schema_editor):
+    """Set relevant Group Permissions for OFA System Admin group."""
+    ofa_system_admin, _ = (
+        # TODO: Merge this with the migration generated for:
+        # https://github.com/raft-tech/TANF-app/pull/1228
+        apps.get_model('auth', 'Group')
+            .objects
+            .get_or_create(name='OFA System Admin')
+    )
+
+    # OFA System Admin has all permissions except delete to all models
+    system_admin_permissions = (
+        apps.get_model('auth', 'Permission')
+            .objects
+            .exclude(delete_permissions_q)
```

Comment:
Note that not even System Admin gets delete permissions. Our models are set up in a way that this is a destructive action and deletion would likely put us in non compliance for certain regulations. The only user that can delete anything in a Cloud.gov environment is the user defined in the `DJANGO_SU_NAME` environment variable - or Lauren by default.